spec/system/candidate_interface/entering_details/international_candidate_enters_their_other_qualification_spec.rb

```diff
@@ -142,14 +164,17 @@ def then_i_see_the_other_qualification_review_page
     expect(page).to have_current_path(candidate_interface_review_other_qualifications_path)
   end
 
-  def and_i_should_see_my_qualification
-    expect(page).to have_content('Master Rules')
-    expect(page).to have_content('Believing in the Heart of the Cards')
+  def and_i_should_see_my_two_qualifications
+    expect(page).to have_content('Master Rules Believing in the Heart of the Cards')
     expect(page).to have_content('2015')
+    # expect(page).to have_content('Master Rules Believing in the Heart of the Cards part II')
```

Comment:
Expectation commented out here.

For this method, I think it would be worth scoping our expectations to the specific `.app-summary-card` containers on the page (ie— finding them all on the page, and using `within` on each one in the array)

Reply:
Could pair that with the [within_summary_row](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/spec/support/test_helpers/candidate_helper.rb#L448) helper to check specific rows.